### ISSUE TITLE
Delete within a word stops at substring delimiters such as _ or change in case

### ISSUE THREAD

**Issue by lucasrangit** [Microsoft/vscode#46203]
Feature Request:
I want to change the only the substring "MACRO" in `VS_MACRO_1` to "CODE", by placing the cursor at the second underscore, pressing Ctrl-Backspace, and type "CODE" to get, `VS_CODE_1`.

Right now, Ctrl-Backspace deletes everything until the word delimiter (e.g. " ", or "/", etc...).

**Comment by garyking** [comment]
Yes would be nice to have this.

In Sublime, on Mac, Option-Backspace deletes entire words, but Ctrl-Backspace deletes up to the first symbol.

Sublime, first with Option-Backspace, then with Ctrl-Backspace:

![backspace](https://user-images.githubusercontent.com/343940/37675308-5768ddb0-2c4b-11e8-8d36-1ab7a4036b43.gif)

**Comment by alexandrudima** [comment]
Covered by #53470

**Comment by chrmarti** [comment]
This works only after adding `_` as a word separator. And that also works in 1.25. Reopening for clarification.

VS Code version: Code - Insiders 1.26.0-insider (c86cab2211d6dce686e5c89934d915b1e020ba9e, 2018-08-02T05:12:16.245Z)
OS version: Darwin x64 17.7.0

**Comment by alexandrudima** [comment]
@chrmarti starting [with the previous release](https://code.visualstudio.com/updates/v1_25#_subword-support) we have commands for subword navigation and subword deletion. These new commands (improved via the PR from my previous comment), cover the case reported in this issue.

In other words, verification steps:
1. configure the keybinding `{ "key": "ctrl+alt+backspace", "command": "deleteWordPartLeft", "when": "textInputFocus && !editorReadonly" }`
2. have the string `VS_MACRO_1`
3. press `ctrl+alt+backspace`
